pwem/protocols/protocol_extract_coordinates.py

```diff
@@ -388,3 +383,12 @@ def _validate(self):
             errors.append('Input particles do not have alignment information!')
 
         return errors
+
+    def _warnings(self):
+        validateMsgs = []
+        if self.applyShifts.get():
+            validateMsgs.append("Only the integer part of the shifts will be applied in order to avoid " +
```

Comment:
I'd avoid protocol class names:

Only the integer part of the shifts will be applied in order to avoid interpolation. If you are re-extracting particles and want to apply the remaining decimal part of the shifts, ask yes to the advance option 'Were particle shifts applied?' in alignment assign protocol.